laravel/resources/views/inc/standard-alerts.blade.php

```diff
@@ -1,5 +1,5 @@
 @if (session('success'))
-    <div class="row">
+    <div class="row me-2">
```

Comment:
[CLARIFICATION]

Is this by design a `me-2` and not a `mt-2` like for the below `div`s?

Reply:
Yep, mt-2 is correct